v2/support/secure_settings_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <!-- http://garden.zendesk.com -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@zendeskgarden/css-bedrock@7.0" type="text/css">
+</head>
+<body>
+  <h2 class="u-gamma">Hello, World!</h2>
+  <!-- https://github.com/zendesk/zendesk_app_framework_sdk -->
+  <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+  <script>
+    // Initialise the Zendesk JavaScript API client
+    // https://developer.zendesk.com/apps/docs/apps-v2
+    var client = ZAFClient.init();
+    client.invoke('resize', { width: '100%', height: '200px' });
+    var settings = {
+      url: 'https://api.thecatapi.com/v1/images/search?format=json',
+      headers: {"x-api-key": "{{setting.catering_token}}"},
+      secure: true,
+      type: 'GET',
+      contentType: 'application/json'
+    };
+    client.request(settings).then(function(data) { console.log(data); });
```

Comment:
These calls should be wrapped inside `app.registered` callback. For example:
```suggestion
    var client = ZAFClient.init();

    client.on('app.registered', function() {
      var settings = { ... };

      client.invoke('resize', { width: '100%', height: '200px' });
      client.request(settings).then(function(data) {
        // console.log(data);
      });
    });
```